plenum/server/replica.py

```diff
@@ -185,6 +186,75 @@ def clear_below_view(self, view_no):
                 del self._batches[v]
 
 
+class ConsensusDataHelper:
```

Comment:
I'm okay with having this class for a while, but I suspect its functionality will be merged into OrderingService

Reply:
that's most probable